applications/luci-app-vpnbypass/Makefile

```diff
@@ -3,42 +3,11 @@
 
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=luci-app-vpnbypass
-PKG_VERSION:=1.0.0
-PKG_RELEASE:=3
-PKG_LICENSE:=GPL-3.0+
-PKG_MAINTAINER:=Stan Grishin <stangri@melmac.net>
```

Comment:
You might leave the maintainer line here. Some newer LuCI apps have the maintainer listed, especially in this kind of case when you are really doing both the main app and the LuCI app.